app/src/main/java/com/goayo/debtify/model/Group.java

```diff
@@ -102,16 +105,16 @@ public void removeUser(Set<User> removeUsers) {
 
     /**
      * Creates a debtTracker and adds it to the list of debtTrackers.
-     *
-     * @param lender        the user who lends out money
+     *  @param lender        the user who lends out money
```

Comment:
this space right here, mr officer

Reply:
With the great power beset on me by the all mighty intellij, the order is returned and the space... well let's just say I sent it to a farm upstate.